src/detect-content.c

```diff
@@ -453,6 +453,22 @@ void SigParseRequiredContentSize(
  */
 bool DetectContentPMATCHValidateCallback(const Signature *s)
 {
+    bool has_pcre = false;
+    bool has_other = false;
+    for (SigMatch *sm = s->init_data->smlists[DETECT_SM_LIST_PMATCH]; sm != NULL; sm = sm->next) {
+        if (sm->type == DETECT_PCRE) {
+            has_pcre = true;
+        } else {
+            has_other = true;
+        }
+    }
+    if (has_pcre && !has_other && s->init_data->sm_cnt == 1) {
+        SCLogWarning("signature id %d uses pcre on raw stream", s->id);
```

Comment:
I would like to remove this warning, and move it to the rule analyzer instead

Reply:
What is the rule analyzer ? Is it not only run with `--engine-analysis` ?

I want fuzzing on rules + pcap to skip these bad rules by erroring on it, so that oss-fuzz can find other timeouts

Reply:
yes, talking only about the `SCLogWarning` statement. Analyzer is `--engine-analysis`.

Reply:
Oh, so we already have `warn_pcre_no_content` in --engine-analysis